extensions/quick-access-for-zeroheight/.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "singleQuote": false,
+  "trailingComma": "all",
+  "semi": true,
+}
```

```suggestion
{
  "printWidth": 120,
  "singleQuote": false
}
```
Let's use the default values